src/Escrow.sol

```diff
@@ -220,6 +223,32 @@ contract Escrow {
         emit Deposit(msg.sender, receiver, amount);
     }
 
+    /**
+     * @dev Deposits escrow for multiple receivers.
```

Comment:
nit: I think this should be `@notice`, and the ones marked as `@notice` below should be `@dev` (https://docs.soliditylang.org/en/latest/natspec-format.html#tags)

Reply:
Agree, I copied the format from the `deposit` function. Looking at the entire code it doesn't seem to be a consistent usage of notice/dev, I think we should fix that but maybe on a separate PR.

Reply:
Okay, I added that as #72. We can address that separately and resolve this conversation here.